src/pages/[title].tsx

```diff
@@ -7,4 +8,22 @@ const NewsFull: NextPage = () => {
   )
 }
 
-export default NewsFull
\ No newline at end of file
+export default NewsFull
+
+export const getServerSideProps: GetServerSideProps = async (context) => {
```

Comment:
please define it before component. So when we reading - first we read what data we fetching and than where we passing this data(component)